cli/unregister/unregister.go

```diff
@@ -56,18 +56,28 @@ func DoIt(forceUnregister, removeNodeUnregister bool, deepClean bool, timeout in
 		}
 
 		// deep clean if anax failed to do it
-		if deepClean && unregErr != nil {
-			if err := DeepClean(); err != nil {
-				fmt.Println(err.Error())
+		if unregErr != nil {
+			if deepClean {
+				if err := DeepClean(); err != nil {
+					fmt.Println(err.Error())
+				} else {
+					unregErr = nil
+				}
 			} else {
-				unregErr = nil
+				msgPrinter.Printf("You may want to run 'hzn unregister -D' to do more clean up.")
```

Comment:
Please change the wording of the message to:
"The node was not successfully unregistered, please use 'hzn unregister -D' to ensure the node is completely reset."
... in both places

Reply:
Done.